rc_lib/view_utils/title_sequence.py

```diff
@@ -0,0 +1,38 @@
+from manim import *
+from typing import List
+
+from rc_lib import style
+
+__all__ = ["TitleSequence"]
+
+
+class TitleSequence:
+    """
+    A class which provides a utility for displaying sequential titles in a step-by-step animation.
+    """
```

Comment:
Could be more descriptive. What is a sequential title / what does this class actually help with? How do you go about using it?